source/beamline/BioXAS/BioXASBeamline.h

```diff
@@ -66,7 +61,7 @@ class BioXASBeamline : public CLSBeamline
 	static BioXASBeamline *bioXAS()
 	{
 		if (!instance_) {
-			instance_ = new BioXASBeamline("BioXAS Beamline");
+			instance_ = new BioXASBeamline("BioXASBeamline", "BioXAS Beamline");
```

Comment:
Based on your answer to @helfrij just below, I'd suggest the name be `BioXAS` (other beamlines being the same).  Otherwise, this short name will only ever be one space shorter than the long name which adds no value.